README.md

```diff
@@ -1,166 +1,153 @@
-<!-- BEGIN MODULE HOOK -->
+## ICD Elasticsearch Module
 
-<!-- Update the title to match the module name and add a description -->
-# Terraform Modules Template Project
-<!-- UPDATE BADGE: Update the link for the following badge-->
-[![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
+[![Graduated (Supported)](https://img.shields.io/badge/Status-Graduated%20(Supported)-brightgreen)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
+[![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-icd-elasticsearch?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/releases/latest)
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
-[![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-module-template?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-module-template/releases/latest)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-<!-- Remove the content in this H2 heading after completing the steps -->
+This module implements an instance of the IBM Cloud Databases for Elasticsearch service.
 
-## Submit a new module
+<!-- BEGIN OVERVIEW HOOK -->
+## Overview
+* [Terraform IBM ICD Elasticsearch Module](#ICD)
+* [Examples](./examples)
+    * [Basic example](./examples/basic)
+    * [Complete example](./examples/complete)
+* [Contributing](#contributing)
 
-:+1::tada: Thank you for taking the time to contribute! :tada::+1:
+<!-- END OVERVIEW HOOK -->
```

Comment:
this doesn't look right ^ Did you use the pre-commit hook to generate this?

Reply:
@ocofaigh No I updated manually

Reply:
Can you pull in the latest version of common-dev-assets (`git submodule update --remote --merge`) and run `pre-commit run --all-files` and it will auto populate this section

Reply:
@Khuzaima05 you didnt update common-dev-assets as per above comment - so its still wrong. Please update

Reply:
@ocofaigh Its updated.
<img width="918" alt="Screenshot 2023-09-07 at 2 58 13 PM" src="https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/assets/56439894/ddf6c718-e8b5-4b47-9ddb-5b4e23ad717c">